portfolio/src/main/webapp/SlideHandler.js

```diff
@@ -0,0 +1,69 @@
+/** The class handling all slides. */
+export default class SlideHandler {
+  
+  /** 
+   * @static @private @const @type {integer} All slides have 5 pages, 
+   * so the max index is 4. 
+   */
+  static MAX_PAGE_INDEX_ = 4;
+
+  constructor(document) {
+    /** @private @type {integer} Index of the page slide 1 is showing. */
+    this.slide1_ = 0;
+    /** @private @type {integer} Index of the page slide 2 is showing. */
+    this.slide2_ = 0;
+    /** @private @const @type {HTML document} */
```

Comment:
make this type non-nullable. https://google.github.io/styleguide/jsguide.html#jsdoc-nullability

Reply:
Get it!